### PR TITLE
Remove p8-platform 

### PR DESCRIPTION
**Pull request by phunkyfish**
7.0.1
- Use std::thread
- Update Queue event to condition_variable/mutex, use chrono and thread::sleep_for
- Remove StringUtils and unrequired p8-platform headers and use std::mutex
- Remove p8-platform dependency
- Remove p8-platform dependency
- Use different socket implementation
- Switch to kissnet for TCP Sockets

### CODE REVIEW

File: src/ClientInstance.cpp [hunk 1]
```diff
@@ -686,7 +686,7 @@ PVR_ERROR CVNSIClientInstance::GetAvailableRecordings(kodi::addon::PVRRecordings
     tag.SetPlotOutline(tag.GetEpisodeName());
     tag.SetPlot(vresp->extract_String());
     tag.SetDirectory(vresp->extract_String());
-    tag.SetRecordingId(StringUtils::Format("%i", vresp->extract_U32()));
+    tag.SetRecordingId(kodi::tools::StringUtils::Format("%i", vresp->extract_U32()));
```
Comment (AlwinEsch) [inline src/ClientInstance.cpp]: Thing for here a `std::to_string(vresp->extract_U32())` can be enough and `StringUtils` no more needed.

Comment (phunkyfish) [reply src/ClientInstance.cpp]: Cool

**Comment by phunkyfish** [comment]
Can go in once you are happy @AlwinEsch

**Review by AlwinEsch — APPROVED** [review]
I'm happy :grinning:

**Pull request merged**